lib/modules/subredditManager.js

```diff
@@ -115,6 +115,19 @@ module.options = {
 		value: true,
 		description: 'Show last update information on the front page (work only if you have at least 50/100 subscription, see <a href="/r/help/wiki/faq#wiki_some_of_my_subreddits_keep_disappearing.__why.3F">here</a> for more info).',
 	},
+	trackVisitedSubreddit: {
+		type: 'boolean',
+		value: true,
+		description: 'Mark subreddits visited when you visit a post listing, comments page, wiki, etc.',
```

Comment:
This might be interpreted as adding to history, maybe "Store the last visited time..."